charts/grafana/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: grafana
-version: 6.40.4
+version: 6.42.0
```

This feature is not sufficient to advance two minor versions:
```suggestion
version: 6.41.0
```